src/pages/index.js

```diff
@@ -32,71 +87,145 @@ const cardPopOut = new PopupWithImage({
 
 cardPopOut.setEventListeners();
 
-function createCard(item) {
-  const card = new Card(item, "#localeCard", openModalImage);
-  return card.generateCard();
+const deleteCardModal = new PopupWithDelete({
+  popupSelector: ".modal_type_delete-image",
+  handleBtnClick: () => {
+    api.deleteCard(cardToDelete.data._id);
+    cardToDelete.deleteCard();
+    deleteCardModal.close();
+  },
+});
+
+/*.then(cardToDelete.deleteCard())*/
+
+function handleDeleteClick(card) {
+  console.log(card);
+  deleteCardModal.open();
+  deleteCardModal.setEventListeners();
```

Comment:
`.setEventListeners()` should be removed from `openings` because it's a **public** method that should be called only **once** in `index.js` otherwise you add listeners again and again to the same elements. This can cause a **`memory leak`**